restli-server/src/main/java/com/linkedin/restli/internal/server/model/RestLiAnnotationReader.java

```diff
@@ -1978,6 +1980,21 @@ private static void addResourceMethods(final Class<?> resourceClass,
     validateResourceModel(model);
   }
 
+  static int getMethodIndex(Method method)
+  {
+    if (method.getAnnotation(BatchFinder.class) != null)
+    {
+      return 1;
```

Comment:
Any reason we do 1 instead of 0? I haven't written a comparator in forever

Reply:
Doesn't matter, just need them in increasing order. I just liked 1-2-3 better.